{{cookiecutter.repository_name}}/debug/train.py

```diff
@@ -0,0 +1,5 @@
+from guild.commands.run import run
+
+
+if __name__ == "__main__":
+    run(["train", "-y", "n_workers=0", "n_batches_per_epoch=2", "--debug-sourcecode=."])
```

Comment:
Small `max_epochs` and `batch_size` as well? I normally can't fit full batch on my laptop when I'm prototyping + it runs faster